x-pack/plugin/watcher/qa/common/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'elasticsearch.build'
```

Comment:
I introduced a qa/common project to better match how the other projects have this configured.

Reply:
Fair enough, but I don't see the point in the additional `testJar`. We're literally just defining another jar that's identical to the default jar. Why another artifact? Can't we just depend on this project via the default configuraiton?

Reply:
thanks... fixed on https://github.com/elastic/elasticsearch/pull/61802/commits/667ea2abd20fc50c91e8ac16f5066ed44aad0d9d